### PR TITLE
Add lens and frames selection page

### PR DESCRIPTION
**Pull request by JessicaJHee**
Kapture 2023-10-10 at 15 16 46

TODOS:
- [x] Confirm lens material options
- [x] Confirm refraction index options
- [x] Add refraction index explaination

### CODE REVIEW

File: frontend/src/pages/visualizeOptionsPage.js
Comment (o-farag) [inline frontend/src/pages/visualizeOptionsPage.js:1]: Shouldn't this button also open the frame selection dialogue?
`<Button variant='filled'>Change Selection</Button>`

**Comment by o-farag** [comment]
@JessicaJHee once I click generate, there's no option to go back
image

I have to click on the view button and then change selection to press the back button from there
image

image

2 thoughts I have:
 I don't see a reason for one page to have just one button. I think the generate button should either 
1. open the popup on the same page or
2. take you to another page with no pop up, the whole page is dedicated to the viewer. It would also have a similar header and title
image

also a few other nitpicks but I think they can be addressed in a different PR

**Comment by JessicaJHee** [comment]
@o-farag 
The next page hasn't been implemented yet, so atm it does nothing and has no logic (to go back).

Once I implement the next page in another PR, it will look like this: 
image
 
I think it makes sense to open another page since there will be 2 options to view the result and also display the weight information. (Also may need an intermediate loading page since it might take a while to generate the models)

**Review by o-farag — APPROVED** [review]
Great work as usual :) Lgtm

**Pull request merged**